data/conf/nginx/includes/site-defaults.conf

```diff
@@ -7,26 +7,43 @@
 
   ssl_protocols TLSv1.2 TLSv1.3;
   ssl_prefer_server_ciphers on;
-  ssl_ciphers ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305;
+  #only supporting strong ciphers  
+  ssl_ciphers ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384;
+  #prioritize ChaCha ciphers  
+  ssl_conf_command Options PrioritizeChaCha;
   ssl_ecdh_curve X25519:X448:secp384r1:secp256k1;
   ssl_session_cache shared:SSL:50m;
   ssl_session_timeout 1d;
   ssl_session_tickets off;
 
-  add_header Strict-Transport-Security "max-age=15768000;";
+  #includeSubDomains; and preload; if all of your services are using HTTPS
+  add_header Strict-Transport-Security "max-age=63072000; includeSubDomains; preload";
```

Comment:
includeSubDomains and preload will essentially lock you out if you are running any non-HTTPS services on the same domain, so that certainly doesn't belong into the default configuration. That was already discussed on a few previous pull request.

Reply:
I agree with @mkuron, the `preload` should be used by host owner explicitly and shouldn't be distributed as a part of default config.